### PR TITLE
Fixed a bunch of comments in the Pro geotrellis analyses

### PR DESCRIPTION
**Pull request by danscales**
Fixed a bunch of comments in the Pro geotrellis analyses
 - Fixed up some really old comments to be more accurate
 - Removed comments about diff geometries in FCD, which we don't use anymore
 - Added various other clarifying comments, including where combine methods are defined.

### CODE REVIEW

**Review by jterry64 — APPROVED** [review]
Thank you for the comments! Especially for the hash partitioner, I always forget what it's doing and the code is hard to follow

**Comment by danscales** [comment]
> Thank you for the comments! Especially for the hash partitioner, I always forget what it's doing and the code is hard to follow

Sure, glad the comments are useful.  Thanks for the review!
